src/config/dynamic-theme-fixes.config

```diff
@@ -24005,6 +24005,60 @@ nav .menu-icon .i
 
 ================================
 
+usosweb.*.edu.pl
+usosweb.*.pl
+usosweb.*.*.pl
+usos.*.*.pl
+web.usos.*.edu.pl
+
+INVERT
+.icons-fontcolor
+.schedimg
+.rejestracja-ikona
+#titlebar > #close:hover
+usos-module-link-tile-container > usos-module-link-tile > h2
+timetable-entry
+
+CSS
+:root {
+    --background: var(--darkreader-neutral-background) !important;
+    --background-secondary: var(--darkreader-bg--background-secondary) !important;
+    --font-color: var(--darkreader-neutral-text) !important;
+}
+#page-body {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+usos-module-link-tile:hover {
+    background-color: var(--darkreader-border--border) !important;
+}
+#close:hover {
+    background-color: var(--darkreader-text--on-primary) !important;
+}
+#close:focus {
+    background-color: #888 !important;
+}
+#layoutCopyright {
+    color: var(--darkreader-neutral-text) !important;
+}
+div > #icon {
+    background: rgb(94, 136, 213) !important;
+}
+timetable-day {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+timetable-entry {
+    color: #111 !important;
+}
+#titlebar {
+    color: var(--darkreader-neutral-text) !important;
+    background-color: var(--darkreader-neutral-background) !important;
+}
+body {
+    background: #333 !important;
```

Comment:
I'm afraid the colour like `#333` may look not good with light colour scheme. Could you check if something like `background: ${#dddddd} !important;` will work? (where `#dddddd` is preferred to be a source colour).

Reply:
Source color is around `#e8e8e8`, but with it, the outer background color is sandwiched between left and right panels (in vanilla, the brightness goes right > left > outer. With this color, it goes right < outer < left). Meanwhile `${#dddddd}` blends too closely with the left panel (only 2 units of difference for me, compared to 8 in vanilla).
I found `${#d8d8d8}` to do the trick and be rather close to the vanilla style, while still looking decent in the light color scheme.